File: domain/src/main/java/no/nav/melosys/domain/mottatteopplysninger/data/FoedestedOgLand.java
package no.nav.melosys.domain.mottatteopplysninger.data;

public class FoedestedOgLand {
    private String foedested;
    private String foedeland;

    public FoedestedOgLand() {}

    public FoedestedOgLand(String foedested, String foedeland) {
        this.foedested = foedested;
        this.foedeland = foedeland;
    }

    public String getFoedested() {
        return foedested;
    }

    public String getFoedeland() {
        return foedeland;
    }
}

File: domain/src/main/kotlin/no/nav/melosys/domain/mottatteopplysninger/data/FoedestedOgLand.kt
package no.nav.melosys.domain.mottatteopplysninger.data

import com.fasterxml.jackson.annotation.JsonCreator
import com.fasterxml.jackson.annotation.JsonProperty

data class FoedestedOgLand @JsonCreator constructor(
    @JsonProperty("foedested")
    val foedested: String,
    @JsonProperty("foedeland")
    val foedeland: String
)
